packages/sfextract.vm/sfextract.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>sfextract.vm</id>
-    <version>2.1.0</version>
+    <version>2.1.0.20240331</version>
     <authors>Joery Droppers</authors>
     <description>command-line utility to extract files from single file bundles in .NET</description>
     <dependencies>
       <dependency id="common.vm" />
-      <dependency id="dotnet-6.0-sdk" version="[6.0.400, 6.1)" />
```

Comment:
We use this syntax on purpose so that this dependency is not updated by our bot. We want to install a single version of dotnet that all our tools use, but we don't really care which one exactly, as I do not expect dotnet minor versions to break our tools. By allowing several version, we decrease the change of a conflict (for example if a dotnet version was already installed and make our updates PR less noisy.

Reply:
Yes, but shouldn't we say install to a known static version ? As for example `6.1` don't exist for dotnet-6-0.

Reply:
It means `>= 6.0.400 and < 6.1`. So basically, allow minor updates from current version as we don't expect libraries like this to break tools in minor updates. Using a concrete version has shown to be too noisy in the updates and it causes issues if you have tools requiring different minor versions. I'll check we have this documented (I think it is not).

Reply:
Okay, I will remove this change from the PR